### ISSUE TITLE
Pre-training language model

### ISSUE THREAD

**Issue by zongxindu** [tricktreat/locate-and-label#10]
I was really inspired by your insight into nested entity recognition, but I have a question: why didn't the paper try to use a large pre-training model(BERT) as token presentation?
